### PR TITLE
Fixes/report show

### LINKED ISSUE #16: reports#show is raising a NoMethodError
When you try to view a report (using reports#show) then the app is raising a `NoMethodError`:

```
2020-11-15T15:27:37.789909+00:00 heroku[router]: at=info method=GET path="/2" host=skunk.fastruby.io request_id=e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9 fwd="69.253.239.22" dyno=web.1 connect=1ms service=20ms status=500 bytes=1827 protocol=https
2020-11-15T15:27:37.785233+00:00 app[web.1]: I, [2020-11-15T15:27:37.785139 #4]  INFO -- : [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9] Completed 500 Internal Server Error in 9ms (ActiveRecord: 1.5ms | Allocations: 883)
2020-11-15T15:27:37.787598+00:00 app[web.1]: F, [2020-11-15T15:27:37.787529 #4] FATAL -- : [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9]
2020-11-15T15:27:37.787598+00:00 app[web.1]: [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9] NoMethodError (undefined method `/' for nil:NilClass):
2020-11-15T15:27:37.787599+00:00 app[web.1]: [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9]
2020-11-15T15:27:37.787600+00:00 app[web.1]: [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9] app/helpers/reports_helper.rb:31:in `stddev_percentage'
2020-11-15T15:27:37.787600+00:00 app[web.1]: [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9] app/controllers/reports_controller.rb:69:in `block in show'
2020-11-15T15:27:37.787601+00:00 app[web.1]: [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9] app/controllers/reports_controller.rb:63:in `each'
2020-11-15T15:27:37.787601+00:00 app[web.1]: [e95f0567-a6fb-4142-8dc4-6b4ee6fe23f9] app/controllers/reports_controller.rb:63:in `show'
```

### PR DESCRIPTION
**Pull request by etagwerker**
Hey, 

This PR fixes #16 and it adds some CSS to display skunk data in its page. It will look like this: 

Screen Shot 2020-11-15 at 9 19 16 PM

Please check it out and let me know if you have any questions. 

Thanks!

### CODE REVIEW

**Review by arielj — COMMENTED** [review]
Hey, I added some comments, these are just suggestions, code looks good

File: app/views/reports/show.html.erb [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
+        <% AnalyzedModule::KEYS.each do |value| %>
+          <th><%= value.humanize %></th>
+        <% end %>
+      </tr>
+    </thead>
+    <tbody>
+    <% @report.data.each do |part| %>
+      <tr class="<%= cycle("odd", "even") %>">
+        <% AnalyzedModule::KEYS.each do |value| %>
+          <td class="<%= cell_class(value) %>">
```
[the remark this answers is not in the document]
Comment (etagwerker) [reply app/views/reports/show.html.erb:14]: @arielj Thanks for your suggestion and good idea! I'm going to use something similar, but not quite exactly what you suggest. 😄

**Comment by etagwerker** [comment]
@arielj I just addressed your comments. 👍 

Let me know if it is better now.

**Review by arielj — APPROVED** [review]
looks good!

**Pull request merged**